app/src/androidTest/java/com/sdpteam/connectout/profile/ProfileFirebaseDataSourceTest.java

```diff
@@ -44,14 +44,6 @@ public void fetchesAllGivenProfiles() {
         assertThat(p2.getName(), anyOf(is(obtained.get(0).getName()), is(obtained.get(1).getName())));
     }
 
-    @Test
-    public void fetchNullProfileWithWrongId() {
-        ProfileFirebaseDataSource model = new ProfileFirebaseDataSource();
-        Profile p = fJoin(model.fetchProfile(UUID.randomUUID().toString() + "NotAPossibleId"));
-
-        assertNull(p);
-    }
-
```

Comment:
The deletion of this test surprised me, could you explain why did you delete it ? I have trouble understanding why its not relevant anymore

Reply:
Like look at the fetchProfiles method : 

List<Profile> profiles = new ArrayList<>();
        allTasks.addOnCompleteListener(dataSnapshots -> {
            for (DataSnapshot dataSnapshot : dataSnapshots.getResult()) {
                Profile profile = dataSnapshot.getValue(Profile.class);
                if(profile != null) {
                    profiles.add(profile);
                }
            }
            futures.complete(profiles);
        });
        
Inside, the problem is that if we have a list of 7 users with incorrect ids, and 3 with correct ids, the profiles returned would be a list of 7 null values and 3 profile objects

Reply:
Thanks for your explication !